app/src/main/java/at/bitfire/icsdroid/ui/EditCalendarActivity.kt

```diff
@@ -225,7 +223,7 @@ class EditCalendarActivity: AppCompatActivity() {
         }
     }
 
-    private fun dirty(): Boolean = subscriptionSettingsModel.dirty() || credentialsModel.dirty()
+    private fun dirty(): Boolean = subscriptionSettingsModel.dirty() || true
```

Comment:
Is this still correct?

Are you going to remove the X icon in this PR or another one?

Reply:
I was thinking of changing it in the PR for the rewrite of `EditCalendarActivity`

Reply:
Okay, everything alright then